source/_integrations/lg_thinq.markdown

```diff
@@ -98,6 +100,67 @@ A read-only property which has only two states that can be toggled is represente
 | | Enable Sabbath mode |
 | [Wine Cellar](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Wine-Cellar/) | Enable Sabbath mode |
 
+#### Sensor
+
+A common read-only property is represented as a sensor platform.
+
```

Comment:
It looks like sometimes, the name listed here is string ID or something technical like that. This is not the idea of the user facing documentation. The idea is to show the name of the sensor, as the user sees it in the frontend, together with additional information explaining what this is, where needed. Also, providing the unit, where applicable.

Reply:
@c0ffeeca7 
We referred to other integrations.
Some of them are not listing up properties at all,
and some of them are just listing up technical names like us.
That's why we list up like these.

for example> smartthings

Capability | Attribute | On-Value
-- | -- | --
accelerationSensor | acceleration | active
contactSensor | contact | open
filterStatus | filterStatus | replace

Reply:
I modified as much as I can as you suggested.